### PR TITLE
Add lot of super link to readme

### PR DESCRIPTION
**Pull request by maximebeaudoin**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ A collection of interesting articles and tools related to development...
 
 - [Storing hundreds of millions of simple key-value pairs in Redis](http://instagram-engineering.tumblr.com/post/12202313862/storing-hundreds-of-millions-of-simple-key-value)
 - [MySQL, Redis, and a Billion Rows - A Love Story](http://www.justincarmony.com/blog/2011/05/23/mysql-redis-and-a-billion-rows-a-love-story/)
+- [Handling 1 Billion requests a week with Symfony2] (http://labs.octivi.com/handling-1-billion-requests-a-week-with-symfony2/)
```
Comment (dominicmartineau) [inline readme.md]: @maximebeaudoin Il y a un espace entre le titre et le lien.

File: readme.md [hunk 2]
```diff
@@ -28,6 +28,9 @@ A collection of interesting articles and tools related to development...
...
+- [NOSQL DATA MODELING TECHNIQUES](https://highlyscalable.wordpress.com/2012/03/01/nosql-data-modeling-techniques/)
```
Comment (dominicmartineau) [inline readme.md]: @maximebeaudoin No caps :)

File: readme.md [hunk 3]
```diff
@@ -39,4 +42,49 @@ A collection of interesting articles and tools related to development...
 
 - [Substance](http://substance.io/) (Substance is a JavaScript library for web-based content editing. It provides building blocks for realizing custom text editors.)
 
-## CSS
\ No newline at end of file
+## CSS
+
+## SSL
+
+### Service
```
Comment (dominicmartineau) [inline readme.md]: @maximebeaudoin Rename it "Tools"

File: readme.md [hunk 4]
```diff
@@ -39,4 +42,49 @@ A collection of interesting articles and tools related to development...
...
+
+- [SSL certificat testing](https://www.ssllabs.com/ssltest/analyze.html)
+
+## DNS
+
+### Service
```
Comment (dominicmartineau) [inline readme.md]: @maximebeaudoin Rename it "Tools"

File: readme.md [hunk 5]
```diff
@@ -39,4 +42,49 @@ A collection of interesting articles and tools related to development...
...
+
+- [DNS Check](http://www.dnsinspect.com/)
+
+## CI
+
+### Services
```
Comment (dominicmartineau) [inline readme.md:61]: @maximebeaudoin Rename it "Tools"

File: readme.md [hunk 6]
```diff
@@ -39,4 +42,49 @@ A collection of interesting articles and tools related to development...
...
+
+- [SSL certificat testing](https://www.ssllabs.com/ssltest/analyze.html)
+
+## DNS
+
+### Service
+
+- [DNS Check](http://www.dnsinspect.com/)
+
+## CI
```
Comment (dominicmartineau) [inline readme.md]: @maximebeaudoin Rename it "Continuous Integration"

File: readme.md [hunk 7]
```diff
@@ -39,4 +42,49 @@ A collection of interesting articles and tools related to development...
...
+
+## GIT
+
+### Articles
+
+- [How to Rebase a Pull Request](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request)
+
+## Debug and testing
+
+### Services
```
Comment (dominicmartineau) [inline readme.md]: @maximebeaudoin Rename it "Tools"

**Comment by maximebeaudoin** [comment]
@dominicmartineau ALL you comments are outdated !!!

**Comment by dominicmartineau** [comment]
@maximebeaudoin hahahaha

**Pull request merged**